compiler/tflchef/proto/tflchef.proto

```diff
@@ -400,7 +404,7 @@ message Operation {
   // LSHProjectionOptions 155
   // SVDFOptions 156
   // RNNOptions 157
-  // L2NormOptions 158
+  optional L2NormOptions l2norm_options = 158;
```

Comment:
I think it would be better to insert `_` between l2 and norm, but I think we will fix it later, so further PRs will not be blocked by this one.